ext/pcre/tests/preg_match_error2.phpt

```diff
@@ -6,11 +6,9 @@ Test preg_match() function : error conditions - wrong arg types
 /*
 * Testing how preg_match reacts to being passed the wrong type of subject argument
 */
```

Comment:
This test seems to be a ZPP test, and should be removed.

Reply:
Done